src/editfns.c

```diff
@@ -628,22 +628,6 @@ If POS is nil, the value of point is used for POS.  */)
 }
 
 
-/* Save current buffer state for `save-excursion' special form.
-   We (ab)use Lisp_Misc_Save_Value to allow explicit free and so
-   offload some work from GC.  */
-
```

Comment:
Take the following two lines and copy them to lisp.h, add `extern` before it and then the compilation will be happy.

Reply:
`lisp.h` already has that declaration: `extern Lisp_Object save_excursion_save (void);`